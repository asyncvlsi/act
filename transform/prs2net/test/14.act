defproc foo(bool x)
{
   bool en, de, a0, a1, b0, b1, c0, c1, _d0, _d1;

   prs {
     en & de -> @x-
   [keeper=0] ~@x & a1 -> _d0-
   }
}

foo f;
