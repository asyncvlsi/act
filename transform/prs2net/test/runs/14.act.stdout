*
*---- act defproc: foo<> -----
* raw ports: 
*
.subckt foo
*.POWER VDD Vdd
*.POWER GND GND
*.POWER NSUB GND
*.POWER PSUB Vdd
*
* --- node flags ---
*
* __d0 (combinational)
*
* --- end node flags ---
*
M0_ #3 en GND GND nch W=0.15U L=0.06U
M1_ #2 de #3 GND nch W=0.15U L=0.06U
M2_ __d0 a1 #2 GND nch W=0.15U L=0.06U
.ends
*---- end of process: foo<> -----
